packages/adminrouter/extra/src/includes/metrics-http-level.conf

```diff
@@ -0,0 +1,15 @@
+# Output fine-grained http directive level metrics using the VTS module.
+# 
+# Caution:
+# A request which matches a location that has a VTS module filter defined will be
+# counted on the Nginx server and location directive level filter as well.
+#
+# This only exists in order for the Nginx VTS module to calculate the average latency
+# per second over all requests for us. This could potentially be removed once we
+# calculate the average latency ourselves in Grafana.
+# 
+# By supplying no value to http= and all= the keys will only be used by our custom
+# Prometheus config in the dcos-monitoring package to relabel the metric __name__
+# but will result in empty labels http and all. Empty labels are dropped by
+# Prometheus and thus they will not show up in Grafana.
+vhost_traffic_status_filter_by_set_key http= ,all=,;
```

Comment:
Is it significant that there is a space after `http=` but not after `all=`? If there is, can we document the reason. If not, can we make them the same, so I don't ask this question another time?

Reply:
Yes, the reason is that `http=` is the value and `,all=,` is the key. They are separated in every filter directive by a space :+1:

Reply:
OK, I did notice that it occurred elsewhere, but I guess it looks most confusing here. And the comma at the start of the key doesn't help. I think this is just one of those cases where I should lookup the directive to understand its format before deciding what it means. So, no change required.